addons/account/models/account_move.py

```diff
@@ -2999,6 +2999,14 @@ def _compute_has_reconciled_entries(self):
         for move in self:
             move.has_reconciled_entries = len(move.line_ids._reconciled_lines()) > 1
 
+    @api.depends('company_id')
+    def _compute_display_qr_code(self):
+        for record in self:
+            if self.move_type not in ('out_invoice', 'out_receipt'):
+                record.display_qr_code = False
+            else:
+                record.display_qr_code = record.company_id.qr_code
```

Comment:
```suggestion
            record.display_qr_code = (
                self.move_type in ('out_invoice', 'out_receipt') 
                and record.company_id.qr_code
            )
```

Reply:
@william-andre thank you for the suggestion, I updated the PR